intermediate_source/model_parallel_tutorial.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """
-Model Parallel Best Practices
-*************************************************************
+1. Model Parallel Best Practices
+================================
 **Author**: `Shen Li <https://mrshenli.github.io/>`_
 
 Data parallel and model parallel are widely-used in distributed training
```

Comment:
It's my bad to mention data parallel in the first sentence in this tutorial. This might confuse people and make them think this tutorial is talking about data parallel. No, this tutorial only talks about model parallel.